code/edge/src/main/java/com/adobe/marketing/mobile/ExperienceEvent.java

```diff
@@ -42,6 +53,33 @@ public Builder() {
 			didBuild = false;
 		}
 
+		/**
+		 * Sets the Datastream identifier to override the default Datastream identifier set in the Edge configuration.
+		 *
+		 * @param datastreamIdOverride Datastream identifier to override the default Datastream identifier set in the Edge configuration
+		 * @return instance of current builder
+		 * @throws UnsupportedOperationException if this instance was already built
+		 */
+		public Builder setDatastreamIdOverride(final String datastreamIdOverride) {
+			throwIfAlreadyBuilt();
+			experienceEvent.datastreamIdOverride = datastreamIdOverride;
+			return this;
+		}
+
+		/**
+		 * Sets configuration to override default Datastream configurations.
+		 *
+		 * @param datastreamConfigOverride configuration map to override default Datastream configurations with
```

Comment:
Suggest rewording: "Map defining datastream configuration overrides for this Experience Event"